BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -128,3 +128,24 @@
     {% endspaceless %}
 {% endblock form_help %}
 
+{% block widget_attributes %}
+{% spaceless %}
+        id="{{ id }}" name="{{ full_name }}"
+        {% if read_only %} readonly="readonly"{% endif %}
```

Comment:
you should still use the {%- if read_only -%} to avoid a lot of white spaces in the generated html